jwst/ramp_fitting/ramp_fit_step.py

```diff
@@ -162,10 +165,13 @@ def create_optional_results_model(opt_info):
         weights=weights,
         crmag=crmag)
 
+    opt_model.meta.filename = input_model.meta.filename
+    opt_model.update(input_model)  # ... and add all keys from input
```

Comment:
I think that calling .update will include the copying of meta.filename from the input to the output model, so line 168 may not even be necessary. Also, do we really want to set the input and output file names to be the same? Shouldn't the opt_model get saved to a file name that at least has a different file type suffix? Or is that handled later?

Reply:
That's done later in the Step code:

https://github.com/spacetelescope/jwst/blob/8a8f148d4ab6310282ac76162890d44b81122840/jwst/ramp_fitting/ramp_fit_step.py#L240-L243